src/content/docs/ci-insights/flaky-test-detection.mdx

```diff
@@ -0,0 +1,240 @@
+---
+title: Flaky Test Detection
+description: Set up automated flaky test detection to identify unreliable tests in your CI pipeline and improve test suite reliability.
+---
+
+import { Image } from "astro:assets"
+
+Flaky test detection helps you identify tests that produce inconsistent results
+on the same code, allowing you to improve the reliability of your test suite
+and reduce false positives in your CI pipeline.
+
+## Understanding Flaky Tests
+
+A **flaky test** is one that has different conclusions on the same SHA1. For
+example, if a test runs twice on the same commit and once fails while the other
+succeeds, it's considered flaky because the outcome is not consistent with the
+same code.
+
+Flaky tests are problematic because they:
+
+- Create false positives that block legitimate deployments
+- Reduce confidence in your test suite
+- Waste developer time investigating non-issues
+- Can mask real bugs when they fail intermittently
+
+## How Flaky Detection Works
+
+Flaky test detection works by running your test suite multiple times on the
+same commit. When tests produce different results (pass/fail) across these runs
+on identical code, they are flagged as flaky. This approach helps identify
+tests that are unreliable and may cause false positives or negatives in your CI
```

Comment:
It seems a bit redundant to mention a third time the reliability and false positives ?

Reply:
Pedagogy is about repetition :)